ext/snowcrash/src/MSONParameterParser.h

```diff
@@ -118,17 +118,19 @@ namespace snowcrash
                             valueMemberSM = typeSection.sourceMap.elements().collection.at(i).value;
                         }
 
-                        if (valueMember)
-                            if (valueMember->valueDefinition.values.size() == 1) {
-                                out.node.values.push_back(valueMember->valueDefinition.values[0].literal);
+                        if (!valueMember)
+                            break;
```

Comment:
`break`? I mean, there should be `continue`, because next element can be correctly *value member*
Also this should be after loop begin (nearly line 115)